packages/assets-controllers/src/AssetsContractController.test.ts

```diff
@@ -56,31 +61,39 @@ const TEST_ACCOUNT_PUBLIC_ADDRESS =
  */
 async function setupAssetContractControllers({
   options,
-  useNetworkControllerProvider,
+  useNetworkControllerProvider = false,
   infuraProjectId = '341eacb578dd44a1a049cbc5f6fd4035',
 }: {
-  options?: Partial<ConstructorParameters<typeof AssetsContractController>[0]>;
+  options?: Partial<
+    Omit<ConstructorParameters<typeof AssetsContractController>[0], 'messenger'>
+  >;
```

Comment:
All constructor options except for `messenger`, which is defined by this function.